rules-reviewed/eap7/eap6/resteasy.windup.xml

```diff
@@ -0,0 +1,826 @@
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <link title="JAX-RS API" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/ext/ReaderInterceptor.html" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000028">
+            <when>
+                <javaclass references="org.jboss.resteasy.spi.interception.MessageBodyWriterInterceptor" />
+            </when>
+            <perform>
+                <hint title="Deprecated MessageBodyWriterInterceptor interface" effort="1" severity="mandatory">
+                    <message>Use XXX instead of `org.jboss.resteasy.spi.interception.MessageBodyWriterInterceptor`.
+                    The Resteasy interceptor facility introduced in release 2.x is replaced by the JAX-RS 2.0 compliant interceptor facility in release 3.0.x.</message>
+                    <link title="RESTeasy SPI Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <link title="JAX-RS API" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/ext/ReaderInterceptor.html" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000029">
+            <when>
+                <javaclass references="org.jboss.resteasy.spi.interception.PostProcessInterceptor" />
+            </when>
+            <perform>
+                <hint title="Deprecated PostProcessInterceptor interface" effort="1" severity="mandatory">
+                    <message>Use XXX instead of `org.jboss.resteasy.spi.interception.PostProcessInterceptor`.
+                    The Resteasy interceptor facility introduced in release 2.x is replaced by the JAX-RS 2.0 compliant interceptor facility in release 3.0.x.</message>
+                    <link title="RESTeasy SPI Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <link title="JAX-RS API" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/ext/ReaderInterceptor.html" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000030">
+            <when>
+                <javaclass references="org.jboss.resteasy.spi.interception.PreProcessInterceptor" />
+            </when>
+            <perform>
+                <hint title="Deprecated MessageBodyWriterInterceptor interface" effort="1" severity="mandatory">
```

Comment:
Wrong `<hint title="...">` and `<message>` -- this rule is about `PreProcessInterceptor`